runtime/precompiles/src/lib.rs

```diff
@@ -34,8 +35,9 @@ pub type MoonbeamPrecompiles<Runtime> = (
 	Ripemd160,
 	Identity,
 	Modexp,
-	// Bn128Add,
-	// Bn128Mul,
-	// Bn128Pairing,
+	Sha3FIPS256,
+	Bn128Add,
+	Bn128Mul,
+	Bn128Pairing,
```

Comment:
Probably want the bn ones to come first so they are at the same addresses as https://github.com/ethereum/go-ethereum/blob/3c46f557/core/vm/contracts.go#L69